commodore/compile.py

```diff
@@ -115,7 +109,14 @@ def _local_setup(config, cluster_id):
             continue
         click.echo(f" > {c}")
         repo = git.init_repository(c)
-        config.register_component(c.name, repo)
+        component = Component(
+            name=c.name,
+            repo=repo,
+            version='master',
+            repo_url=None,
```

Comment:
It would be possible to configure the repo_url here (probably a good idea to do so, in light of the pending local mode rewrite/improvements), by setting `repo_url=repo.remotes.origin.url`.